incubator/dokuwiki/values.yaml

```diff
@@ -0,0 +1,63 @@
+## Bitnami dokuwiki image version
+## ref: https://hub.docker.com/r/bitnami/dokuwiki/tags/
+##
+image: bitnami/dokuwiki:20160626a-r0
+
+## Specify a imagePullPolicy
+## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+## User of the application
+## ref: https://github.com/bitnami/bitnami-docker-dokuwiki#environment-variables
+##
+dokuwikiUsername: superuser
```

Comment:
All bitnami applications name the default user as `user`. I think we should follow the same approach in this chart/image as well.